survey-sdk/src/main/java/co/nimblehq/survey/sdk/entity/SurveyEntity.kt

```diff
@@ -40,3 +41,11 @@ data class SurveyEntity(
     val questionList: List<QuestionEntity>?
         get() = questions?.get(document)
 }
+
+fun SurveyEntity.toSurveyModel(): SurveyModel {
```

Comment:
now you can mark everthing inside this sdk (class, fun, const) as `internal`. You can update all in a separated PR later solely for the sdk changes, just make sure that you have a ticket to keep track

Reply:
@sleepylee Yup, I will create a ticket for keep tracking.